README.md

```diff
@@ -75,6 +75,10 @@ Then you can run the migrations
 - `SERVER_NAME` : optional, for redirects only
 - `SLAASK_WIDGET_KEY` : optional, for visitors chat with you
 
+## Enable Postgis on PostgreSQL
```

Comment:
This duplicates postgis setup.
Either your note should be at l42, or you should move l42..49 here.
I'm not sure about any requiered order between involved steps, but I'd rather keep the existing order.